.github/workflows/main.yml

```diff
@@ -97,7 +97,7 @@ jobs:
               -DCMAKE_CXX_COMPILER=clang++-10 -DCMAKE_C_COMPILER=clang-10
               -DCMAKE_CXX_FLAGS="-ffile-prefix-map=${{ runner.workspace }}/=/
                 -fno-omit-frame-pointer -fsanitize=address,undefined
-                -fsanitize-undefined-strip-path-components=$UBSAN_STRIP_COUNT"
+                -fsanitize-undefined-strip-path-components=$UBSAN_STRIP_COUNT -Og"
```

Comment:
Nit: I'd move it as the first flag instead of burying it after all the debugging/sanitizers related junk.